Run.hx

```diff
@@ -1,13 +1,244 @@
-import peg.*;
+import peg.PegException;
 
 class Run {
+	static var basePackage = '';
+
+	static function getType(t:peg.php.PType):String {
+		return switch t.getName() {
+			case 'TInt':
+				'Int';
+			case 'TFloat':
+				'Float';
+			case 'TString':
+				'String';
+			case 'TArrayOfString':
+				'Array<String>';
+			case 'TBool':
+				'Bool';
+			case 'TArray':
+				'Array<Dynamic>';
+			case 'TObject':
+				'Map<String,Dynamic>';
+			case 'TCallable':
+				'Dynamic';
+			case 'TMixed':
+				'Dynamic';
+			case 'TResource':
+				'php.Resource';
+			case 'TClass':
+				var c = t.getParameters().shift();
+				switch (~/^\\/.replace(c, '')) {
+					case 'stdClass':
+						'php.StdClass';
+					// These are in the haxe standard library
+					case 'ArrayAccess' | 'Closure' | 'Error' | 'ErrorException' | 'Exception' |
+					'Generator' | 'IteratorAggregate' | 'RuntimeException' |
+					'SessionHandlerInterface' | 'StdClass' | 'Throwable' | 'Traversable':
+						'php.${c}';
+					// These are in the haxe standard library
+					case 'Mysqli' | 'Mysqli_driver' | 'Mysqli_result' | 'Mysqli_stmt' |
+					'Mysqli_warning' | 'PDO' | 'PDOException' | 'PDOStatement' | 'SQLite3' |
+					'SQLite3Result' | 'SQLite3Stmt':
+						'php.db.${c}';
+					// These are in the haxe standard library
+					case 'ReflectionClass' | 'ReflectionFunctionAbstract' | 'ReflectionMethod' |
+					'ReflectionProperty' | 'Reflector':
+						'php.reflection.${c}';
+					// These require phpnatives (https://lib.haxe.org/p/phpnatives/)
```

Comment:
Unfortunately phpnatives groups php toplevel types into custom packages. That doesn't follow current scheme in haxe std lib for php.
It's ok to rely on phpnatives for now, but in future this have to be changed.
No need to change anything now, just making a note.